repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Flow.scala

```diff
@@ -46,7 +46,7 @@ object Flow {
     */
   def process[F[_]: Logger: Timer: ConcurrentEffect](resources: Resources[F])(events: Stream[F, Model.Record[F, EventContainer]]): Stream[F, Unit] = {
     val inserting = events
-      .evalMap(checkAndInsert[F](resources.bigQuery, resources.env.config.datasetId, resources.env.config.tableId, resources.backoffTime))
+      .parEvalMapUnordered(resources.concurrency)(checkAndInsert[F](resources.bigQuery, resources.env.config.datasetId, resources.env.config.tableId, resources.backoffTime))
```

Comment:
Out of curiosity, I've already seen `.chunkN()` being used before using `.parEvalMap`, is there an interest in doing so?

Reply:
I _think_ we might want to do that if we wanted to buffer events and send them in microbatches. But here we want to fire `checkAndInsert` as soon as possible, with just a single event and in parallel, so chunking looks a bit redundant.